packages/style/scss/elements/btn.scss

```diff
@@ -249,6 +249,50 @@
     &.collapsible-header {
         border-radius: 0;
     }
+
+    &.mod-loading {
+        &.mod-danger .loading-spinner,
+        &.mod-link .loading-spinner,
+        &.mod-confirm .loading-spinner,
+        &.mod-confirm-error .loading-spinner,
+        &.mod-confirm-success .loading-spinner {
+            display: none;
+        }
+
+        &:disabled .loading-spinner:before,
```

Comment:
is that necessary since the bg and border are white by default?

Reply:
It's not really necessary, you're right